osu.Game/Overlays/SkinEditor/SkinEditorOverlay.cs

```diff
@@ -147,7 +147,7 @@ public void SetTarget(OsuScreen screen)
 
             if (skinEditor == null) return;
 
-            skinEditor.Save();
+            skinEditor.Save(false);
```

Comment:
If we're doing optional boolean params again then could I at least ask for a

```suggestion
            skinEditor.Save(userTriggered: false);
```